README.md

```diff
@@ -48,95 +100,375 @@ Options:
-      rest of the doc
-  velocity:  # field
-    doc: |
-        A single block of doc string.
-```
-
-This structure appears in Nxdl as follows:
+# NeXus groups in YAML format
+source_TYPE(NXsource):
+  exists: recommended
+  doc: |
+    A source used to generate a beam.
+(NXmanipulator):
+  exists: optional
+  doc: |
+    Manipulator for positioning of the sample.
+  value_log(NXlog):
+    exists: optional
+```
 
-```xml
-<field name="energy">
-    <doc>
-          Part1 of the entire doc.
-          part1 of the entire doc.
+### NeXus Field and NeXus Attrubute
+NeXus group may contain NeXus fields, NeXus attributes, and NeXus groups. A field, that does not have preceding `NX`, and a attribute, preceded by `\@`, must have a [NeXus type](https://manual.nexusformat.org/nxdl-types.html#index-0) (e.g.`NX_FLOAT`, `NX_CHAR`). In YAML format each NeXus field or NeXus attribute has a implicit type `NX_CHAR` otherwise type must be denoted inside the paranthesis (e.g. `end_time(NX_DATE_TIME)`). Other xml attributes of NeXus `field` and NeXus `attribute` comes as children of the field and attribute (the special keywords will be discussed on next section). The introductory text of the NeXus fields or attributes goes under `doc` child.
 
-          This concept is related to term `&lt;term&gt;`_ of the &lt;spec&gt; standard.
+A NeXus group may encompass multiple `field`, `attribute`, and subgroup, each serving distinct purposes within the data structure. The [`field`](https://manual.nexusformat.org/design.html#design-fields), denoted without the prefix NX, and [`attribute`](https://manual.nexusformat.org/design.html#design-attributes), indicated by `\@`, must be associated with a NeXus type (e.g., `NX_FLOAT`, `NX_CHAR`). In YAML format, each field or attribute (NeXus attribute) implicitly assumes the type `NX_CHAR`, unless explicitly specified within parentheses (e.g., `end_time(NX_DATE_TIME)`).
 
-          .. _&lt;term&gt;: &lt;url&gt;
+Additionally, `XML` attributes specific to NeXus field and attribute are represented as children of the corresponding `field` or `attribute` elements (further details on special keywords will be discussed in the following section). Descriptive information pertaining to NeXus `field`s or `attribute`s is encapsulated within the `doc` child element.
 
-          Rest of the doc
-          rest of the doc
-    </doc>
-    <field name="velocity">
-        <doc>
-              A single block of doc string.
-        </doc>
-    </field>
-</field>
+**NeXus field and attribute in YAML format**
+```yaml
+(NXentry):
+  exsits: required
+  definition:  # Field type: NX_CHAR
+    \@version:  # Attribute type: NX_CHAR
+    enumeration: [NXmpes]
+  title:
+  start_time(NX_DATE_TIME):  # Field type: NX_DATE_TIME
+    doc: Datetime of the start of the measurement.
+  end_time(NX_DATE_TIME):  # Field type: NX_DATE_TIME
+    exists: recommended
+    doc: Datetime of the end of the measurement.
 ```
 
-### Additional Keywords
+### NeXus Link
```

Comment:
Thanks for notifying it. If NAIC comes to a decision, then we can remove this part. Is it fine? Because we have many application using the link feature.

Reply:
Ok, let's just keep it for now.